governator-test-junit/src/main/java/com/netflix/governator/guice/test/AnnotationBasedTestInjectorManager.java

```diff
@@ -48,20 +51,19 @@
     private final SettableConfig classLevelOverrides = new DefaultSettableConfig();
```

Comment:
We reset class and method configs separately. We only reset the test config after individual tests.